### PR TITLE
Dev - Improved unicode fort name handling

### PR DESCRIPTION
**Pull request by marksweb**
Short Description: Improved the handling of logged unicode characters & general code quality improvements.

An improvement of https://github.com/PokemonGoF/PokemonGo-Bot/commit/190640f06fc623f804c554ce1ac9264c9fa8b22d

### CODE REVIEW

**Comment by pokecunt** [comment]
download

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -13,18 +14,19 @@
...
-from human_behaviour import sleep
-from item_list import Item
-from metrics import Metrics
 from pokemongo_bot.event_handlers import LoggingHandler, SocketIoHandler
 from pokemongo_bot.socketio_server.runner import SocketIoRunner
-from worker_result import WorkerResult
-from tree_config_builder import ConfigException, TreeConfigBuilder
+from . import cell_workers
+from . import logger
+from .api_wrapper import ApiWrapper
```
Comment (DayBr3ak) [inline pokemongo_bot/__init__.py]: why change the imports ?

File: pokemongo_bot/__init__.py [hunk 2]
```diff
@@ -243,7 +245,7 @@ def check_session(self, position):
 
     @staticmethod
     def is_numeric(s):
-        try:
+        try:
```
Comment (DayBr3ak) [inline pokemongo_bot/__init__.py]: trailing space

**Comment by douglascamata** [comment]
I got a very big PR changing the whole logging system, so I'll close this.

**Pull request closed without merging**